src/ars/adapters/outbound/http.py

```diff
@@ -33,32 +34,30 @@
 class ClaimValidity(BaseModel):
     """Start and end dates for validating claims."""
 
-    class Config:
-        extra = "forbid"
-        frozen = True
+    model_config = ConfigDict(extra="forbid", frozen=True)
 
     valid_from: DateTimeUTC = Field(
-        ..., description="Start date of validity", example="2023-01-01T00:00:00Z"
+        ..., description="Start date of validity", examples=["2023-01-01T00:00:00Z"]
     )
     valid_until: DateTimeUTC = Field(
-        ..., description="End date of validity", example="2023-12-31T23:59:59Z"
+        ..., description="End date of validity", examples=["2023-12-31T23:59:59Z"]
     )
 
-    @validator("valid_until")
-    @classmethod
-    def period_is_valid(cls, value, values):
+    @model_validator(mode="after")
```

Comment:
Did not know about those kind of validators, probably have to read up on some of the more useful changes coming with pydantic v2.

Reply:
Yes, in deed, these are very nice.